contentcuration/contentcuration/static/js/edit_channel/new_channel/views.js

```diff
@@ -56,6 +57,9 @@ var ChannelList  = BaseViews.BaseEditableListView.extend({
 		$(".disable-on-edit").prop("disabled", edit_mode_on);
 		$(".disable-on-edit").css("cursor", (edit_mode_on) ? "not-allowed" : "pointer");
 		$(".invisible-on-edit").css('visibility', (edit_mode_on)?'hidden' : 'visible');
+		(edit_mode_on)? $(".new_channel_button").addClass("disabled") : $(".new_channel_button").removeClass("disabled");
```

Comment:
https://trello.com/c/Qn9qtdJX/223-consider-hiding-new-channel-button-while-the-editor-is-open